dockstore-webservice/src/test/java/io/dockstore/webservice/languages/NextflowHandlerTest.java

```diff
@@ -0,0 +1,19 @@
+package io.dockstore.webservice.languages;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NextflowHandlerTest {
+
+    @Test
+    public void testGetRelativeImportPathFromLine() {
+        Assert.assertEquals("modules/rnaseq.nf", NextflowHandler.getRelativeImportPathFromLine("include { RNASEQ } from './modules/rnaseq'", "/main.nf"));
+        Assert.assertEquals("modules/rnaseq.nf", NextflowHandler.getRelativeImportPathFromLine("include { RNASEQ } from './modules/rnaseq.nf'", "/main.nf"));
+        // TODO: Replace with Java 12's http://openjdk.java.net/jeps/326
+        Assert.assertEquals("modules.nf", NextflowHandler.getRelativeImportPathFromLine("include { \n"
```

Comment:
Pending Java 12, might be more readable as a resource file. 
But I don't feel strongly about it, I think I can get the general idea here.